src/components/Artists/Artist.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+
```

Comment:
maybe hard to disambiguate. 

check out the `disambiguation` key from the api response